src/molecules/link-card/link-card.test.tsx

```diff
@@ -1,3 +1,59 @@
+import React from "react";
+import faker from "faker";
+import { LinkCard } from "./link-card";
+import { TestUtils } from "../../tests/test-utils";
+import { LinkCardTypes } from "../constants/link-card-types";
+
 describe("LinkCard", () => {
-    test.skip("TODO - https://github.com/AndcultureCode/AndcultureCode.JavaScript.React.Components/issues/36", () => {});
+    test("When default props, renders LinkCard", () => {
```

Comment:
Nitpick - for consistency, we don't usually capitalize test names in JS/TS (references to components/proper nouns are fine)